submissions/kosh3n/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,202 @@
+body {
+    background-color: #dff6ff;
+    background-image: url("data:image/svg+xml,%3Csvg width='64' height='64' viewBox='0 0 64 64' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8 16c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.582 8 8 8zm0-2c3.314 0 6-2.686 6-6s-2.686-6-6-6-6 2.686-6 6 2.686 6 6 6zm33.414-6l5.95-5.95L45.95.636 40 6.586 34.05.636 32.636 2.05 38.586 8l-5.95 5.95 1.414 1.414L40 9.414l5.95 5.95 1.414-1.414L41.414 8zM40 48c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.582 8 8 8zm0-2c3.314 0 6-2.686 6-6s-2.686-6-6-6-6 2.686-6 6 2.686 6 6 6zM9.414 40l5.95-5.95-1.414-1.414L8 38.586l-5.95-5.95L.636 34.05 6.586 40l-5.95 5.95 1.414 1.414L8 41.414l5.95 5.95 1.414-1.414L9.414 40z' fill='%230d97cd' fill-opacity='0.4' fill-rule='evenodd'/%3E%3C/svg%3E");
+    color: #06283D;
+}
+
+header {
+    background-color: #47B5FF;
+    display: flex;
+    justify-content: space-around;
+    height: 90px;
+    overflow: hidden;
+
+}
+
+.homo-img {
+    background-color: #47B5FF;
+    padding-left: 80px;
+}
+
+.welcome {
+    font-size: 2em;
+    padding-left: 300px;
+    padding-right: 100px;
+    text-align: center;
+}
+
+/*menu*/
+
+.navigation {
+    width: 400px;
+}
+
+#hamburger:checked~.nav-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    border: 1px solid #333;
+    background-color: #1363DF;
+    position: absolute;
+    z-index: 4;
+}
+
+/*hide menu*/
+.nav-menu {
+    display: none;
+    list-style-type: none;
+    max-width: 370px;
+    max-height: 500px;
+    overflow-y: scroll;
+    overflow-x: hidden;
+}
+
+/*hide items*/
+
+#show-more:checked~.hide-items {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    background-color: #1363DF;
+    width: 100%;
+}
+
+.hide-items {
+    list-style-type: none;
+    display: none;
+    margin-left: -10px;
+}
+
+.more-btn {
+    text-align: center;
+}
+
+/*hide items end*/
+
+ul {
+    padding: 15px;
+}
+
+/*link styles*/
+
+a {
+    text-decoration: none;
+    color: #06283D;
+    font-size: 26px;
+}
+
+a:hover {
+    opacity: 0.7;
+    background-color: #4D7EFF;
+    color: #fefefe;
+}
+
+/*inputs*/
+
+label {
+    display: inline-block;
+    cursor: pointer;
+    padding-top: 30px;
+    margin-left: 250px;
+}
+
+label:hover {
+    opacity: 0.5;
+}
+
+input {
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)